src/components/Book.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+
+function Book(props) {
+  return (
+    <Wrapper>
+      <Img>
+        <div>
+          <img src={props.bookCover} alt="bookcover" />
+        </div>
+      </Img>
```

Comment:
Do you need all these wrappers around the `img`? Is it not possible to give the styles to the image directly?